packages/docs/src/components/sidebar/Sidebar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <va-sidebar class="sidebar" v-model="visible" :width="sidebarWidth">
+  <va-sidebar class="sidebar" v-model="visible" color="background" :width="sidebarWidth">
```

Comment:
If you set sidebar default color to the background you don't need to add this prop here.
```suggestion
  <va-sidebar class="sidebar" v-model="visible" :width="sidebarWidth">
```